src/lib/circleci.ts

```diff
@@ -44,15 +44,17 @@ export class CircleCI {
    * @returns {Object[]} Projects, as returned by CircleCI API.
    */
   async getProjects() {
-    const repoNameRegex = new RegExp(this.config.repoNameRegex);
+    const repo = this.config.repos[0];
```

Comment:
@alexander-fenster @theacodes so... this is a little janky.  In the next quarter, I think we should drop the circleci support from this tool, or kick it out to it's own module.  Given our migration to kokoro I'm all like 🤷‍♂️

Reply:
I'm totally fine if we just drop all CircleCI logic. It's only used in `repo-check.js` (`.ts`) and it basically just checks that the project is enabled in Circle. We can safely remove that.

Reply:
👍 I'll remove it in another PR then.